qutebrowser/javascript/webelem.js

```diff
@@ -362,6 +362,11 @@ window._qutebrowser.webelem = (function() {
         document.execCommand("insertText", false, text);
     };
 
+    funcs.dispatch_event = (id, event) => {
+        const elem = elements[id];
+        elem.dispatchEvent(new Event(event));
```

Comment:
The [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Events/input) for `InputEvent` say that it bubbles up to other elements, while the [docs](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event) for the `Event()` constructor says that `bubbles` is `false` by default.

I guess we should add `bubbles`, `cancleable` and `composes` (with the proper default values) to the Python side, pass them to JS as well, and pass them to `new Event`.

Reply:
This is a little odd, when running `new InputEvent("input")` `bubbles` is false. Maybe the `bubbles` property is set to true by default when the event is emitted from the browser (so I'll try doing that here as well).